src/components/HomePageContainer.js

```diff
@@ -53,6 +73,20 @@ const HomePageContainer = () => {
     return null;
   }
 
+  let dateToCheck = new Date(userHealthFacility.contractEndDate);
+  const timeDelta = getTimeDifferenceInDaysFromToday(dateToCheck);
+  const getHealthFacilityStatus = (timeDelta) => {
+    if(timeDelta>180){
```

Comment:
I will create a separate const for this '180' and '30'. Btw. it might be good idea to create overall const like:
`
const DAYS = {
DAY: 1,
WEEK: 7,
MONTH: 30,
YEAR: 360,
};
`
If you won't create it like this, consider creating at least constants with more precise name.